docs/baremetal-agent-controller.md

```diff
@@ -136,7 +188,7 @@ spec:
     credentialsName: bmc-secret
 ```
 
-Setting `automatedCleaningMode` field and the `inspect.metal3.io` annotations are both optional. If
+Setting `automatedCleaningMode` field and the `inspect.metal3.io` is optional. If
```

Comment:
Along the lines of my previous comment, what do you think about having the `assisted_deployment` script in `dev-script` prepare/modify the BareMetalHost manifest for us (starting from the one created by dev-scripts)?